### ISSUE TITLE
Recent Comments Page is Flooded by the Notes Posts

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#1514]
The Recent Comments Page is a nice addition. This list of posts is flooded by the Notes Posts, which is making it difficult to find new posts in any other board.

if there was a way to break up the posts so it showed each discussion board in a separate section with a set of recent posts that would be useful, but right now I'm finding it a bit overwhelming to use it to find new discussions that require research team input or of interest.

**Comment by mschwamb** [comment]
Just realized that now each board has a recent comments as well as on the project Talk level. I'm talking about the full project Talk recent comments, which would be the most helpful to research teams and moderators and others who want to quick scan the boards for new conversations and posts

**Comment by vrooje** [comment]
A lot of times I'm specifically looking to see new Notes (without having to click on each thread) when I view the Recents page, so I don't want this to go away.

**Comment by vrooje** [comment]
(Also the board-level recents page doesn't work for me -- it loads recent posts from other Talks and other boards as well. The top-level one seems fine.)

**Comment by mschwamb** [comment]
I like the full post nature it shows as well. I'm only suggesting segmented  it so that one board (the notes board) doesn't drown out the other boards

**Comment by vrooje** [comment]
Yeah, I like the idea of having the *option* to view either Recent Subject Notes and Recent Everything-But-Subject-Notes.

**Comment by mschwamb** [comment]
That could work, but for me I pay attention and reply faster to posts in Help for example, so I'd still like a way of sorting those out if it could be divided by board, but if not - having a Notes included/Notes not included toggle/setting would be helpful

**Comment by parrish** [comment]
(After fixing the board filtering for recents) I'm pretty sure this is as intended.  If you want to exclude notes, then I think just viewing the non-note boards' recents is the way to do that.

Honestly, keeping the subject comments visible is still really important to talk.

**Comment by DarrenMcRoy** [comment]
Keeping them visible is important, sure, but why not offer the option (non-default) to have them filtered out?

**Comment by vrooje** [comment]
Good point @parrish - that is already an option if you view the recents on a by-board basis.

**Comment by parrish** [comment]
Most projects don't have that many boards (average is 1.9 non-subject boards), so filtering by board isn't much of a hassle (I think).

From a software perspective the more special cases we can avoid, the better.

This isn't a set-in-stone opinion either.  My gut is that's

**Comment by DarrenMcRoy** [comment]
Well, you can view the recents on a by-board basis, but is there a way to view "all boards except Notes?"

**Comment by parrish** [comment]
It's a feature that doesn't exist, but could.

> Most projects don't have that many boards (average is 1.9 non-subject boards), so filtering by board isn't much of a hassle (I think).

I mean that in the sense of going to each non-note board recent comments.

**Comment by DarrenMcRoy** [comment]
Right, and I'm saying that's more time-intensive, having to go to multiple places. I'm hoping that we can consolidate all relevant information in a single place (but allow them to filter out the noise that Meg is finding distracting).

**Comment by mschwamb** [comment]
Top level  recent comments as currently set up will always be dominated by the Notes, since they get generated at a higher rate. It's the reason they were separated out by default from the discussion boards and put into Notes so they don't dominate/drown out  the other discussions. I agree keeping the subject/notes  comments visible is important, but  I also think making it as easy as possible for researchers and moderators (especially people that might only check Talk once a day or every few days) to find other discussions is important too. 

I found the recents page of Talk 2.0 extremely useful because it was divided into the separate boards. In one click I could see everything that's going on. I can scan to see if there were new threads in other boards (and see roughly if I wanted to click and comment) , there were 140 character comments (Notes here) to comment on without loosing info about the other discussion boards.  if there were more than one new conversation in the Help board, etc, I could see it without having to click through to check, but could also see the past several 140 character/notes comments.. To do that now, requires multiple clicks to the different boards/different boards recent comments. 

I'm not arguing it has to be done exactly the way it was done previously. I really love that in Panoptes Talk the recents comments page shows the whole post, but separating in some way  the notes on the Talk front page level recent comments, in my opinion would be this capability really really useful to science team and moderators  if this isn't too difficult of a change to implement.

**Comment by mschwamb** [comment]
Should mention, I'm not suggesting get rid of the Notes board posts, just somehow separate out recent comments from non-Notes boards from the Notes board on the new Talk front page recent comments list.

**Comment by vrooje** [comment]
Is this a sufficiently quick task that it's worth making saving us a few
clicks a priority over items where we don't yet have the functionality at
all?

**Comment by mschwamb** [comment]
Apologies if I sounded like I was implying or imposing a priority or schedule. That certainly wasn't my intention. I apologize if it came across that way. I'm only making a suggestion based on my experiences and what I thought would be helpful for other researchers. Whether or not the development team decide to implement it or in what time frame is not something I should be or am attempting to comment on here.

**Comment by parrish** [comment]
Hey, no worries!

It's not a big change, code-wise.  My main concern is simply to not hide away subject comments.  We've heard a lot of complaints in the past versions during user interviews that basically said that users felt that nobody was going find their comments.

I think notifications help with that to an extent, but we've never completely solved the discoverability problem for this.

I think if we did implement this, we'd definitely want to make them visible by default as Darren mentioned.

I'd also like to let the new recent views stand as they are for a while to let people figure out if it's really an issue or not.  It's entirely possible that it's fine as is.

Again, I'm happy to revisit this in a couple weeks after we've got a feel for the new features.  At that point we could poll people on a talk thread as well if you'd like.
